tracing/opencensus.go

```diff
@@ -144,33 +155,36 @@ func WithExporterFull(name, host string, logger *zap.SugaredLogger) ConfigOption
 			exporter trace.Exporter
 			closer   io.Closer
 		)
-		switch cfg.Backend {
-		case config.Zipkin:
-			// If host isn't specified, then zipkin.NewEndpoint will return an error saying that it
-			// can't find the host named ''. So, if not specified, default it to this machine's
-			// hostname.
-			if host == "" {
-				n, err := os.Hostname()
+		if cfg != nil {
```

Comment:
curious why the config can be nil now when before it wasn't.

Reply:
The `Finish/Shutdown` function is passing nil to the config option on shutdown. But so far this function was only called in tests and those tests never used this function `WithExporterFull`. From now on the Shutdown function will be called with the config option provided by WithExportedFull so it needs to check if the passed config is nil to prevent `panic`.